src/main/java/com/coinbase/prime/model/enums/WalletType.java

```diff
@@ -14,7 +14,7 @@
  *  limitations under the License.
  */
 
-package com.coinbase.prime.model.wallets;
+package com.coinbase.prime.model.enums;
```

Comment:
WEB3 is ONCHAIN here now. might want to proactively add QC
export enum WalletType {
  UnknownWalletType = 'UNKNOWN_WALLET_TYPE',
  Vault = 'VAULT',
  Trading = 'TRADING',
  WalletTypeOther = 'WALLET_TYPE_OTHER',
  Qc = 'QC',
  Onchain = 'ONCHAIN',
}